bin/inference/pycbc_inference_plot_posterior

```diff
@@ -129,6 +125,26 @@ if opts.plot_prior is not None:
     prior = option_utils.prior_from_config(cp)
     logging.info("Drawing samples from prior")
     prior_samples = prior.rvs(opts.prior_nsamples)
+    # we'll just use the first file for metadata
+    fp = fps[0]
+    # add the static params
+    for param in fp.attrs['static_params']:
+        setattr(prior_samples, param, fp.attrs[param])
+    # remap any parameters
+    if 'remapped_params' in fp.attrs:
+        remapped_params = {}
+        for func, param in fp.attrs['remapped_params']:
+            try:
+                remapped_params[param] = prior_samples[func]
+            except (NameError, TypeError):
+                continue
+        prior_samples = FieldArray.from_kwargs(**remapped_params)
+        for param in fp.attrs['static_params']:
+            setattr(prior_samples, param, fp.attrs[param])
```

Comment:
@cdcapano Do you have to do repeat this again here? This was done in lines 131 - 132 already, no?

Reply:
Ah, okay I see what you're doing, looking at this again. 
You're re-populating prior_samples for the remapped_params case & not editing what it had before. I did not read the changes correctly last time, sorry.